Cargo.toml

```
@@ -61,6 +61,7 @@ hyper = "1"
 jsonrpsee = "0.24"
 schemars = "0.9"
 tower = "0.4"
+nix = "0.30" # `stop` RPC method
```

Please add dependencies alphabetically within their group.

Also, due to us depending on Zebra which also has this dependency, this should use the same version (0.29).
```suggestion
nix = "0.29" # `stop` RPC method
schemars = "0.9"
tower = "0.4"
```